app/views/posts/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<section class="user-info">
+<div class="image-info">
+
+</div>
+<div class="info">
+ <h2> <%= @user.name %> </h2>
+ <h3>Number of posts: <%= @user.posts_counter %></h3>
+ </div>
+</section>
+
+<section  class="post-info">
+  <% @posts.each do |post| %>
+    <%= link_to user_post_path(@user, post) do %>
+   <div class="post">
+   <h2>Post: <%= '#' %> <%= post.id %></h2>
+    <h3><%= post.title%></h3>
+    <p>Comments: <%= post.comments_counter || 1 %>
+    <span class="span-like">Likes: <%= post.likes_counter  || 0 %></span></p>
+    </div>
+    <div class="post-text">
+    <p><%= post.text%> </p>
+    </div>
+    <% end %>
+    <% end %>
```

Comment:
- [ ] Kindly display recent comments here rather than all comments, as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)